apps/api/package.json

```diff
@@ -24,6 +24,7 @@
     "prisma:migrate:deploy": "prisma migrate deploy",
     "prisma:migrate:deploy-test": "dotenv -e .env.test -- pnpm exec prisma migrate deploy",
     "prisma:generate": "prisma generate dev && pnpm --filter=@codeimage/prisma-models build",
+    "docker:up": "docker compose -f docker-compose.dev.yml up",
```

Comment:
@riccardoperra this is useful to me so as not to always put the complete command, can we leave it?